model/service.go

```diff
@@ -30,6 +31,16 @@ type Service struct {
 	Runtime     Runtime
 	Framework   Framework
 	Node        ServiceNode
+
+	Origin *ServiceOrigin
+}
+
+// ServiceOrigin holds self nested information related to the monitored
+// service.
```

Comment:
Not sure about this. Can we say it's information related to the originating service? e.g.

`// ServiceOrigin holds information about the service that originated a transaction.`